src/main.cc

```diff
@@ -242,185 +152,41 @@ static int daemon_main(LiteClient& client, const bpo::variables_map& variables_m
     return EXIT_FAILURE;
   }
 
-  client.importRootMetaIfNeededAndPresent();
-  client.finalizeInstall();
+  std::shared_ptr<LiteClient> client_ptr{&client, [](LiteClient* /*unused*/) {}};
+  // Setting apply_lock parameter to false, since we already took the FileLock above
```

Comment:
Maybe we should the AkliteClient's lock for sake of consistency and remove the one at line 145.

Reply:
It was done so in the next commit.